docs/requirements.txt

```diff
@@ -1,5 +1,4 @@
 sphinx>=3.0
 sphinx-rtd-theme==0.5.1
 sphinx-autodoc-typehints==1.11.1
-numpy==1.19.5
-pandas==1.2.1
+redata==0.3.0
```

Comment:
Since we recently updated redata to v0.3.1, does it matter to update it across repositories? Perhaps documentation updates shouldn't be tagged to avoid this issue. This way minor updates could be reserved for patches in the code base.

Reply:
You raised a fair point, I had refer to 0.3.0 in part because that's the minimum version in this refactoring. I think the best way to go is to do `>=0.3.0`. This would require fewer dependabot updates, etc and makes it clear what version of `redata` is needed.